server/modules/record/manager/_recordManager/recordUpdateManager.js

```diff
@@ -93,28 +93,57 @@ const deleteRecordsPreview = async (surveyId) =>
  */
```

Comment:
should the sequence diagram be updated as well?

Reply:
of course